crm_compassion/tests/test_crm_compassion.py

```diff
@@ -107,7 +119,7 @@ def _create_event(self, lead, type):
                 'name': event_dico['context']['default_name'],
                 'type': type,
                 'start_date': now,
-                'end_date': now,
+                'end_date': now + ' 8:43:00',
```

Comment:
For what is that for? I think we should replace both start_date and end_date by `fields.Datetime.now()`